src/errors/InvalidElementForSlicingError.ts

```diff
@@ -0,0 +1,5 @@
+export class InvalidElementForSlicingError extends Error {
```

Comment:
I know we haven't really used this functionality much, but... I think it would be good for this to `implements Annotated` and have:
```ts
specReferences = ['http://hl7.org/fhir/elementdefinition-definitions.html#ElementDefinition.slicing'];
```

Reply:
https://github.com/FHIR/sushi/pull/952/commits/6e644f6f9393105bf1b2b6925f49a939c47cbcd7